json/windows-server-2025-64bit.json

```diff
@@ -0,0 +1,11 @@
+{
+    "uuid": "905ab443-e415-4d52-bfef-e14b147eac4e",
+    "reference_label": "windows-server-2025-64bit",
+    "name_label": "Windows Server 2025 (64-bit) (preview)",
+    "name_description": "Clones of this template will automatically provision their storage when first booted and then reconfigure themselves with the optimal settings for Windows Server 2025 (64-bit).",
+    "derived_from": "base-windows-uefi.json",
+    "min_memory": "2G",
```

Comment:
Where did this value and the 64G disk come from?

Reply:
This was from a discussion on Slack with Peter, 2G is the minimum for the desktop version of WS25, the server version can get away with 512M but it would "barely be functional"
64G is a similar story, 32G is the absolute minimum but requires a slightly larger disk as the RAM increases so I was advised to increase this to 64G

Reply:
Server Core can run with 512M, but requires more to install. Server (Desktop) requires 2G

Server Core can run fine on 32G, but extra space for updates should be considered. Server (Desktop) can run (just about) on 32G, but this is a bare minimum, and any practical use should have more storage.

Reply:
OK, thanks for the info.